api/src/domain/updateRoadmapSidebarCards.test.ts

```diff
@@ -98,6 +108,9 @@ describe("updateRoadmapSidebarCards", () => {
     it("adds tax-registration-nudge if formation task is complete", () => {
       const taskId = formationTaskId;
       const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "NEEDS_TO_REGISTER_FOR_TAXES",
+        }),
         taskProgress: { [taskId]: "COMPLETED" },
```

Comment:
i think we should delete lines like this and change the test name because now the taskProgress isn't the field that's determining what happens anymore, right?